src/worker/requirements.txt

```diff
@@ -49,8 +51,9 @@ pyarrow==13.0.0
 pycparser==2.21
 pydantic==2.1.1
 pydantic_core==2.4.0
-pylance
+pylance==0.8.3
```

Comment:
same thing here with`pylance` - please remove the version numbers to avoid a dependency clash